templates/nodered/nodered.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Node-RED graphical event wiring tool
+Wants=network.target
+Documentation=http://nodered.org/docs/hardware/raspberrypi.html
+
+[Service]
+Type=simple
+# Run as normal pi user - change to the user name you wish to run Node-RED as
+User=pi
+Group=pi
+WorkingDirectory=/home/pi
+
+Environment="NODE_OPTIONS=--max_old_space_size=512"
```

Comment:
Where does this value come from? The nodered docs state half the value?

https://nodered.org/docs/getting-started/raspberrypi:

> To do this, you should use the alternative node-red-pi command and pass in the max-old-space-size argument.
> node-red-pi --max-old-space-size=256